specs/interop/derivation.md

```diff
@@ -294,9 +297,10 @@ The expiry window is the time period after which an initiating message is no lon
 ### Depositing an Executing Message
 
 Deposit transactions (force inclusion transactions) give censorship resistance to layer two networks.
-It is possible to deposit an invalid executing message, forcing the sequencer to reorg. It would
+If it were possible to deposit an invalid executing message, this would force the sequencer to reorg. It would
 be fairly cheap to continuously deposit invalid executing messages through L1 and cause L2 liveness
-instability. A future upgrade will enable deposits to trigger executing messages.
```

Comment:
Read this several times and it didn't scan. Seems to contradict itself, so I changed it to what I _think_ we are trying to say here. But please check!

Reply:
The new text is better. Yes this part of the spec was broken, I think along the time the access-list change happened. The new spec fix looks good to me.